charts/timescaledb-single/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.serviceMonitor.enabled }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "timescaledb.fullname" . }}
+  {{- if .Values.serviceMonitor.namespace }}
+  namespace: {{ .Values.serviceMonitor.namespace }}
+  {{- end }}
+  labels:
+    app: {{ template "timescaledb.fullname" . }}
+    chart: {{ template "timescaledb.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    cluster-name: {{ template "clusterName" . }}
+    {{- if .Values.serviceMonitor.labels -}}
+    {{ toYaml .Values.serviceMonitor.labels | nindent 4 }}
+    {{- end }}
+spec:
+  endpoints:
+  - interval: {{ .Values.serviceMonitor.interval }}
+    {{- if .Values.serviceMonitor.scrapeTimeout }}
+    scrapeTimeout: {{ .Values.serviceMonitor.scrapeTimeout }}
+    {{- end }}
+    honorLabels: true
+    port: {{ .Values.serviceMonitor.portName }}
+    path: {{ .Values.serviceMonitor.path }}
+    {{- if .Values.serviceMonitor.metricRelabelings }}
+    metricRelabelings: {{ toYaml .Values.serviceMonitor.metricRelabelings | nindent 6 }}
+    {{- end }}
+  jobLabel: "{{ .Release.Name }}"
+  selector:
+    matchLabels:
+      app: {{ template "timescaledb.name" . }}
```

Comment:
Hi, I am not the chart's maintainer but I tested your PR.

It seems that the Service which you try to target has an `app: {{ template "timescaledb.fullname" . }}` label.
I had to modify it to this value to monitor my deployment so my ServiceMonitor has:

```yaml
  selector:
    matchLabels:
      app: {{ template "timescaledb.fullname" . }}
      release: "{{ .Release.Name }}"
      component: postgres-exporter
```

Reply:
You are right. Adding the label ```component: postgres-exporter```  monitors the svc which has 3 endpoints(3 pods-1 master and 2 replicas). I will make changes to the PR accordingly. Thank you

Reply:
@mshivanna this is the only thing preventing me from merging your PR :) Could you update it?

Reply:
@paulfantom Have fixed it. sorry about the late fix. Changed selector match label from app: {{ template "timescaledb.name" . }} to app: {{ template "timescaledb.fullname" . }}